src/fu-util.c

```diff
@@ -679,6 +686,12 @@ fu_util_install (FuUtilPrivate *priv, gchar **values, GError **error)
 	if (!fwupd_client_install (priv->client, id, filename, priv->flags, NULL, error))
 		return FALSE;
 
+	/* at least one device asked to show a success message */
+	if (priv->current_message != NULL) {
+		g_print ("%s\n", priv->current_message);
+		priv->current_message = NULL;
```

Comment:
then you can just g_free this in the common finalize... Or can we just read it from `fwupd_device_get_update_message (priv->current_device)` here?

Reply:
I intentionally did "caching" the string it the way I did because priv->current_device will change for composite cab files.  If there is a message from any of them it needs to get cached for when done processing the composite CAB.